parlai/core/teachers.py

```diff
@@ -60,9 +60,6 @@ def __init__(self, opt):
         self.num_workers = opt.get('numthreads', 1)
         self.request_queue = queue.Queue()
 
-    def __len__(self):
```

Comment:
was this wrong i guess?

Reply:
yeah, this was old code in the `DataLoader` that is useless now